src/main/java/com/heroiclabs/nakama/DefaultClient.java

```diff
@@ -174,18 +174,18 @@ public SocketClient createSocket(final String host, final int port) {
     }
 
     @Override
-    public SocketClient createSocket(final int port, final int socketTimeoutMs) {
-        return new WebSocketClient(host, port, ssl, socketTimeoutMs, this.trace);
+    public SocketClient createSocket(final int port, final int socketTimeoutMs, final int socketPingMs) {
+        return new WebSocketClient(host, port, ssl, socketTimeoutMs, socketPingMs, this.trace);
     }
 
     @Override
     public SocketClient createSocket(final String host, final int port, final boolean ssl) {
-        return new WebSocketClient(host, port, ssl, 5000, this.trace);
+        return new WebSocketClient(host, port, ssl, 5000, 5000, this.trace);
```

Comment:
nit: Lets also make these values `public final` constants in the SocketClient class.

Reply:
Done.